cg/services/order_validation_service/workflows/tomte/models/sample.py

```diff
@@ -10,9 +10,9 @@ class TomteSample(Sample):
     age_at_sampling: float | None = None
     control: ControlEnum | None = None
     elution_buffer: str | None = None
-    father: str | None = Field(None, pattern=NAME_PATTERN)
+    father: str = Field(pattern=NAME_PATTERN)
     formalin_fixation_time: int | None = None
-    mother: str | None = Field(None, pattern=NAME_PATTERN)
+    mother: str = Field(pattern=NAME_PATTERN)
```

Comment:
I think they are still nullable (a sample may not have a mother/father in the case)